newIDE/app/src/Export/GenericExporters/OnlineWebExport/OnlineGamePropertiesDialog.js

```diff
@@ -153,6 +153,7 @@ export const OnlineGamePropertiesDialog = ({
         setDiscoverable={setDiscoverable}
         displayThumbnail
         thumbnailUrl={thumbnailUrl}
+        disabled={isLoading}
```

Comment:
Disabled the whole form while the game is submitting, for a better UX (otherwise only the button disables)